### ISSUE TITLE
Set up a donations address

### ISSUE THREAD

**Issue by freeatnet** [forkdelta/proposals#7]
Multiple people have reached out offering donations to help the service along. I think that's a sign to set something up.

Questions:
* How are those funds distributed and used?
* Do we reward donations? If so, how?
* Are there any restrictions on donations accepted?

_ADDED_
* What is the spending schedule of the donations?

**Comment by freeatnet** [comment]
My personal opinions are:
* First, a percentage of donations should go towards bounties for development, design, security, and other help. Second, server cost.
* Keep a public list of donors. Perhaps allow donations to have a message attached to them (subject to review).
* No more than 1 ETH from a given address. No donations that clearly track back to tumblers/mixers. Non-conforming donations will be sent back less transaction fees (using ethgasstation SafeLow as gas price).

**Comment by JonathonDunford** [comment]
Agreed.

However, a few things:

I think server costs come first, then bounties for development.

Agreed with second point. Maybe even have a running announcement channel for donations over X ETH.

Why put a cap on donations? 1 ETH is a relatively low denomination for some.

**Comment by freeatnet** [comment]
@JonathonDunford I tend to think of server cost almost as a fixed cost, whereas bounties are a variable cost. More good development = more users = more funding (or sooner move to a sustainable funding stream).

Re: donation cap: I'm trying to avoid skewing donation pool towards one source, disincentivizing smaller donors from giving or creating a notion of large donors potentially receiving some special treatment by the project. We can either adjust the amount or use a percentage of a previous month's donations total to set the limit.

**Comment by JonathonDunford** [comment]
In my opinion, the fixed costs of server upkeep lends itself to being paid first. That way you understand what budget you have left over.

Donors could still file multiple donations for 1 ETH. A donation cap is a bad idea. When I ran a game server, we ran on donations and the people who only were able to make small donations always wanted to become a large donor and would continue to donate small amounts. It wasn't a penalty, it was a goal.

Besides, we don't have rewards or a favor system for donors. The only purpose is to support the project. If the only purpose is to support the project, we need as much help as we can get.
